src/formatters/index.stories.mdx

```diff
@@ -0,0 +1,6 @@
+import { Meta } from "@storybook/addon-docs";
+import FormattersJsDoc from "dist/docs/Formatters.md";
+
+<Meta title="Utilities/Formatters" />
+
+<FormattersJsDoc />
```

Comment:
We do something similar for the release notes page in storybook - the `.mdx` story format allows you to write or import markdown to display as a story. In this case, we import markdown generated from jsDoc comments in `src/formatters`